examples/voice_messages_list.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import messagebird
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+
+
+# ACCESS_KEY = ''
```

Comment:
quite a few other examples read the access key from the command line, maybe we can make that consistent here? :)